### ISSUE TITLE
Open Source License

### ISSUE THREAD

**Issue by dkieffer** [tveye79/calculator#1]
Have you considered making your project open source? I was searching around for a simple Javascript calculator to use as a web app and I found your project here. I was interested in contributing to the project, but would definitely be more likely to if it is open source. Here's an example of a license you could use if that is something you are interested in:

https://www.gnu.org/licenses/agpl-3.0.html

Basically all you'd need to do is add that as a text file just like these folks did with their project MongoDB: https://github.com/mongodb/mongo/blob/master/GNU-AGPL-3.0.txt
